subsys/net/ip/tcp2_priv.h

```diff
@@ -180,3 +184,14 @@ struct tcp { /* TCP connection */
 
 #define FL(_fl, _op, _mask, _args...)					\
 	_flags(_fl, _op, _mask, strlen("" #_args) ? _args : true)
+
+/*
+ * @brief Generate initial TCP sequence number
+ *
+ * @return Return a random TCP sequence number
+ */
+static inline u32_t tcp_init_isn(void)
```

Comment:
Shall we just reuse sys_rand32_get() in place? Currently the TTCN-3 sanity check needs it not randomized, so under #ifdef CONFIG_NET_TEST_PROTOCOL

Reply:
Sure, I will add relevant code.